src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -3535,6 +3535,13 @@ static WellKnownMembers()
                     0,                                                                                                      // Method Signature
                     (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_String, // Return Type
 
+                // System_Runtime_CompilerServices_RequiredMembersAttribute__ctor
```

Comment:
Typo: should be singular "RequiredMember"

Let's update the spec to reflect recent LDM decision. 
[https://github.com/dotnet/csharplang/blob/main/proposals/required-members.md#metadata-representation](https://github.com/dotnet/csharplang/blob/main/proposals/required-members.md#metadata-representation)

Reply:
https://github.com/dotnet/csharplang/pull/5688